docs/git.md

```diff
@@ -62,6 +62,7 @@ These are the names of the branches that are specially treated by the Simperby n
 3. `p`: the block proposal for this node. The node operator may push or force-push to this branch. When pushed, the Git server will check the validity of the branch. The consensus engine will recognize this branch and propose to the consensus. It stands for 'block proposal'.
 4. `a-<number>`: a valid agenda (but not yet approved) propagated from other nodes. If the governance has approved the agenda, it will point to the `agenda-proof` commit which lies on top of the agenda commit. The number is arbitrarily assigned.
 5. `b-<number>`: a valid (but not yet finalized) block propagated from other nodes. The number is arbitrarily assigned.
+6. `fp`: a very special branch that always holds the finalization proof for the last block. This is required because a block header doesn't contain the finalization proof of itself. Thus, to make a repository self-verifiable, it is essential to have the proof somewhere, in some way. This branch has only one empty commit that is directly on top of the genesis commit, titled with `finalization proof of the last block`. The commit message body contains the actual proof.
```

Comment:
Is empty commit a commit with no state transition? am I right?

Reply:
an 'empty commit' is a term from Git (not our original context); a commit that does not have a diff, but only the commit message.

Reply:
understood